docs/getting_started.md

```diff
@@ -13,30 +13,14 @@ Before jumping the gun, we recommend you start with the high-level
 
 #### Installation Requirements:
 
-- Python >= 3.8
-- PyTorch >= 1.11
-- gpytorch == 1.9.1
-- linear_operator == 0.3.0
-- scipy
-- multiple-dispatch
-- pyro-ppl >= 1.8.4
-
 BoTorch is easily installed via
-[Anaconda](https://www.anaconda.com/distribution/#download-section) (recommended)
-or `pip`:
+[Anaconda](https://www.anaconda.com/distribution/#download-section):
 
-<!--DOCUSAURUS_CODE_TABS-->
-<!--conda-->
 ```bash
 conda install botorch -c pytorch -c gpytorch -c conda-forge
 ```
-<!--pip-->
-```bash
-pip install botorch
-```
-<!--END_DOCUSAURUS_CODE_TABS-->
```

Comment:
Hmm there are probably a good amount of folks that don't use anaconda (and maybe aren't able to), and so I'd be hesitant to remove the `pip install` from the getting started page here.